templates/taxi/driver_list.html

```diff
@@ -5,10 +5,16 @@
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
-        <li>
+        {% if driver.username == user.username %}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }} (Me))
```

Comment:
The template correctly appends '(Me)' to the current user's entry, which is a nice touch for user experience.